addons/spreadsheet/static/src/global_filters/plugins/global_filters_core_plugin.js

```diff
@@ -32,8 +32,8 @@ import { escapeRegExp } from "@web/core/utils/strings";
 export class GlobalFiltersCorePlugin extends spreadsheet.CorePlugin {
     constructor(config) {
         super(config);
-        /** @type {Object.<string, GlobalFilter>} */
-        this.globalFilters = {};
+        /** @type {Array.<GlobalFilter>} */
+        this.globalFilters = [];
```

Comment:
Just a question, but why did you change this structure ? I suppose it's to keep a real order, but there is any other reason ?

Reply:
Yeah it's for the order, we cannot have ordering while using a dictionnary